source/core.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: UTF-8 -*-
-#core.py

```

Comment:
Please restore this line

Reply:
@leonardder According to the [copyright headers wiki page](https://github.com/nvaccess/nvda/wiki/Copyright-headers)

> Note: We decided to remove the filename comment from the top of the file, since it doesn't really add anything, and is a source of error on file rename / copying copyright headers between files.

If we arre no longer adding it to the new files why shouldn't we remove it from the existing ones during updating of the copyright headers?